plugins/httpapi/dcnm.py

```diff
@@ -42,6 +42,8 @@ def __init__(self, *args, **kwargs):
         self.headers = {"Content-Type": "application/json"}
         self.txt_headers = {"Content-Type": "text/plain"}
         self.version = None
+        # Retry count for send API
+        self.retrycnt = 5
```

Comment:
Minor but personally feel that we should just use `self.retrycount`.  It's only two extra letters and less ambiguous

Reply:
Changes done as suggested